src/meson.build

```
@@ -15,7 +15,7 @@ dependencies = [
     # Version limitation for GLib.ApplicationFlags.DEFAULT_FLAGS
     dependency('glib-2.0', version: '>= 2.74'),
     # Version limitation for automatic load of Application.css in Granite.init()
```

```suggestion
    # Version limitation for Granite.Bin
```

I decided to write reasons for each version limitations to avoid magic version numbers for later references and for community packages, so I'd like you to update it.